documentation/configure.markdown

```diff
@@ -269,11 +269,7 @@ server. Note that users installing PuppetDB via [the module][module]
 will already have PostgreSQL configured properly and these steps
 should not be necessary.
 
-At a minimum, you will need to ensure that you have PostgreSQL 9.6 or
-later running that will accept incoming connections, a user and
-an empty database for PuppetDB. Information on
-connection/authentication configuration in PostgreSQL and be found
-[here](https://www.postgresql.org/docs/9.6/static/auth-pg-hba-conf.html). Docs
+Ensure that you have PostgreSQL 9.6 running that will accept incoming connections, a user and an empty database for PuppetDB. Information on connection/authentication configuration in PostgreSQL and be found [here](https://www.postgresql.org/docs/9.6/static/auth-pg-hba-conf.html). Docs
```

Comment:
Thought about adjusting this a bit, perhaps something like "...have configured a PostgreSQL 9.6 server to include a user and empty database for PuppetDB, and to accept connections to that database as that user."